.github/workflows/main.yml

```diff
@@ -119,3 +119,16 @@ jobs:
         run: |
           cargo test --locked --manifest-path simplicity-sys/Cargo.toml
           cargo test --locked --workspace --all-features
+
+  docs:
+    name: Docs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Crate
+        uses: actions/checkout@v4
+      - name: Checkout Toolchain
+        uses: dtolnay/rust-toolchain@nightly
```

Comment:
36ff9613917a4b2f339499a9d3d2f96a0c59a319: an unpinned nightly version might lead to flaky CI, but it should be ok for checking if docs build.

Reply:
We already have one :). I figured we could pin both instances at once. (Which we should do!)